package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@dynatrace/opentelemetry-exporter-metrics",
   "description": "OpenTelemetry metrics exporter for Dynatrace",
-  "version": "0.8.0",
+  "version": "0.9.0",
```

Comment:
Do we go to 1.0.0 with this one too?

Reply:
Let's do a prerelease first, and then release 1.0.0 later :slightly_smiling_face: